.circleci/config.yml

```diff
@@ -22,20 +22,34 @@ jobs:
       - run:
           name: Install Dependencies
           command: |
+            pip install --user --upgrade pip neuromation
             pip install --user -r .circleci/requirements.txt
       - run:
           name: Build Image
           command: |
             neuro config login-with-token $NEURO_TOKEN
+            neuro config show
             neuro config docker
             export IMAGE_NAME=neuromation/ml-recipe-midi-generator
             export IMAGE_REF=$IMAGE_NAME:$CIRCLE_BUILD_NUM
             export IMAGE_URI=image:$IMAGE_REF
-            make setup \
-                __BAKE_SETUP=yes \
-                CUSTOM_ENV_NAME=$IMAGE_URI \
-                PROJECT_PATH_STORAGE=storage:build-$CIRCLE_BUILD_NUM
-            neuro pull $IMAGE_URI
+            export PROJECT_PATH_STORAGE=storage:build-$CIRCLE_BUILD_NUM
+            export CUSTOM_ENV_NAME=$IMAGE_URI
+            make setup __BAKE_SETUP=yes | tee
+
+            export PROJECT_PATH_ENV=/midi-generator
+            make -f test.mk test_jupyter_demo TRAINING_MACHINE_TYPE=cpu-small | tee
+            echo -e "\n[+] Test passed: $PROJECT_PATH_ENV on CPU\n"
+            make -f test.mk test_jupyter_demo TRAINING_MACHINE_TYPE=gpu-small | tee
+            echo -e "\n[+] Test passed: $PROJECT_PATH_ENV on GPU\n"
+
+            export PROJECT_PATH_ENV=/project-local
+            make -f test.mk test_jupyter_demo TRAINING_MACHINE_TYPE=cpu-small | tee
+            echo -e "\n[+] Test passed: $PROJECT_PATH_ENV on CPU\n"
+            make -f test.mk test_jupyter_demo TRAINING_MACHINE_TYPE=gpu-small | tee
+            echo -e "\n[+] Test passed: $PROJECT_PATH_ENV on GPU\n"
```

Comment:
Perhaps, for testing baked jupyter, it's better not to mount anything and run as-is. Will remake.

Reply:
done